.travis.yml

```diff
@@ -33,6 +36,14 @@ script:
  - cabal test --show-details=always
  - cabal sdist
  - cabal haddock | grep "100%" | wc -l | grep "4"
+ - if [[ "$TRAVIS_OS_NAME" != "osx" ]]; then docker build .; fi
+
+deploy:
```

Comment:
@javcasas I think it would be better to build the images directly in Docker Hub https://docs.docker.com/docker-hub/builds/#create-an-automated-build. We could use GH tags to trigger builds in Docker Hub.

Reply:
Yup, I think it's better to build on the Docker hub.